e2e/README.md

```diff
@@ -1,19 +1,9 @@
-## Multus e2e test with kind
-
-### How to test e2e
-
-
-```
+How to test e2e
 $ git clone https://github.com/k8snetworkplumbingwg/multus-cni.git
 $ cd multus-cni/e2e
 $ ./get_tools.sh
 $ ./generate_yamls.sh
 $ ./setup_cluster.sh
 $ ./test-simple-macvlan1.sh
-```
-
-### How to teardown cluster
-
-```
-$ ./teardown.sh
-```
+How to teardown cluster
```

Comment:
This change seems not to related license, so could you please revert it?
(you may change it in #934)